source/localizable/feedspec/index.el.html.md.erb

```diff
@@ -628,6 +630,236 @@ To χρώμα του προϊόντος. Το πεδίο χρώματος λει
+αναγνωρίζονται από το <%= settings.site_name.capitalize %>:
+
+<div class="alert alert-danger">
+  <p>
+    Τα size variations μπορούν να χρησιμοποιηθούν μόνο για προϊόντα που
+    διατίθενται σε διαφορετικά μεγέθη.
+  </p>
+</div>
+
+<div class="alert alert-danger">
+  <p>
+    Η HTML <strong>δεν</strong> επιτρέπεται σε κανένα από τα πεδία των size variations.
+  </p>
+</div>
+
+Όνομα                                                  | Τύπος   | Μέγιστο Μήκος                                        | Υποχρεωτικό | Δοκιμαστικό<b><sup>1</sup></b> | Παράδειγμα
+------------------------------------------------------ | ------- | ---------------------------------------------------- | ----------- | ------------------------------ | -------------
+[Variation Unique ID](#variation-unique-id)            | String  | <span class="mono">200</span>                        | Ναι         | Όχι                            | `12431.XL`
+[Variation Availability](#variation-availability)      | String  | <span class="mono">100</span>                        | Ναι         | Όχι                            | `Σε 2 μέρες`
+[Variation Size](#variation-size)                      | String  | <span class="mono">64</span>                         | Ναι         | Όχι                            | `M`
+[Variation Quantity](#variation-quantity)              | Integer | <span class="mono">8</span>                          | Ναι         | Όχι                            | `10`
+[Variation Link](#variation-link)                      | String  | <span class="mono">2000</span>                       | Όχι         | Ναι                            | `https://www.someurl.co.uk/products/322.html`
+[Variation Price](#variation-price)                    | Decimal | <span class="mono">- (up to 2 decimal places)</span> | Όχι         | Ναι                            | `23.22`
+[Variation MPN](#variation-mpn)                        | String  | <span class="mono">80</span>                         | Όχι         | Ναι                            | `GF322234`
+[Variation EAN / Barcode](#variation-ean--barcode)     | Integer | <span class="mono">80</span>                         | Όχι         | Ναι                            | `9780471117094`
+
+<small><b><sup>1</sup></b> Tα Δοκιμαστικά (Beta) πεδία προτείνεται να εισαχθούν στα size variations.
+Ωστόσο οι τιμές τους δεν θα υιοθετηθούν αρχικά αλλά Θα χρησιμοποιηθούν οι τιμές
+απο τα αντίστοιχα πεδία του προϊόντος (Product).</small>
+
+#### Variation Unique ID
+
+To πεδίο αυτό είναι το **μοναδικό** αναγνωριστικό του
+size variation στο κατάστημά σας. **Η τιμή του πεδίου αυτού είναι υποχρεωτικό να παραμείνει
+ίδια καθ' όλη τη διάρκεια "ζωής" του size variation**.
+
+> Το Unique ID δεν μπορεί να υπάρχει σε **παραπάνω από ένα size variation**. Αν υπάρχει
+παραπάνω από μία φορά μέσα στο feed, τότε το size variation θα αποθηκευθεί με τα
+στοιχεία της τελευταίας σε σειρά εμφάνισης.
```

Comment:
I think there is a mistake here, both on products and variations. I talked with @vinPopulaire and we think that we will always read the first occurrence. In any case we need to correct this on both product and variation documentation once we find the answer.

Reply:
I fixed it in both product and variation.